content/blog/getting-started-with-hpe-greenlake-api-for-data-services.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Getting started with HPE GreenLake API for Data Services
+date: 2024-04-02T00:37:09.418Z
+priority: 0
+author: Ron Dharma
+authorimage: /img/rondbust.jpg
+thumbnailimage: /img/alletra-element-small.png
+disable: false
+---
+## What’s New?
+
+Recently, a new set of REST APIs for GreenLake Edge to Cloud Platform was introduced on the [HPE GreenLake Developer website](https://developer.greenlake.hpe.com/docs/greenlake/services/data-services/public/) at https://developer.greenlake.hpe.com/docs/greenlake/services/data-services/public/.  These APIs are grouped under the set which is called HPE GreenLake API for Data Services.
```

Comment:
Well, I turned it around so that we group up the API as HPE GreenLake API for Data Services. The challenge is that engineering choose this common APIs as "HPE GreenLake for Data Services" Truth to be told if developer.greenlake.hpe.com accept it, it will be fine. I just like to call every APIs today as HPE GreenLake APIs because eventually all services will use the

Reply:
single token and common end-points.